examples/text/document_classification_20newsgroups.py

```diff
@@ -83,7 +83,12 @@
               help="Remove newsgroup information that is easily overfit: "
                    "headers, signatures, and quoting.")
 
-(opts, args) = op.parse_args()
+
+def is_interactive():
+    return not hasattr(sys.modules['__main__ '], '__file__')
+
+argv = [] if is_interactive() else sys.argv[1:]
```

Comment:
Maybe add a comment somewhere saying that this is an IPython console or Jupyter notebook work-around